### PR TITLE
Add test for command and control, single shot mode

### PR DESCRIPTION
**Pull request by ctron**

### CODE REVIEW

**Comment by kornys** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by kornys** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by ctron** [comment]
Rebased.

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared-iot

File: systemtests/src/main/java/io/enmasse/systemtest/utils/TestUtils.java [hunk 1]
```diff
@@ -563,4 +570,37 @@ public static void waitForChangedResourceVersion(final TimeoutBudget budget, fin
             }
         }, budget);
     }
+
+    public static void waitForAddressSpaceReady(final String namespace, final String name, final TimeoutBudget budget) throws Exception {
```
Comment (famartinrh) [inline systemtests/src/main/java/io/enmasse/systemtest/utils/TestUtils.java]: The methods added to this class are never used, delete it please

Comment (famartinrh) [reply systemtests/src/main/java/io/enmasse/systemtest/utils/TestUtils.java]: I'll do that, plus other minor changes

Comment (ctron) [reply systemtests/src/main/java/io/enmasse/systemtest/utils/TestUtils.java]: @famartinrh please wait … I just pushed a new commit that makes use of it

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared-iot

File: systemtests/src/main/java/io/enmasse/systemtest/utils/TestUtils.java (hunk 1 above)
Comment (famartinrh) [reply systemtests/src/main/java/io/enmasse/systemtest/utils/TestUtils.java]: @ctron there is already a method for checking if address space is ready `io.enmasse.systemtest.utils.AddressSpaceUtils#waitForAddressSpaceReady`

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared-iot

File: systemtests/src/main/java/io/enmasse/systemtest/utils/TestUtils.java (hunk 1 above)
Comment (ctron) [reply systemtests/src/main/java/io/enmasse/systemtest/utils/TestUtils.java]: I didn't know that :smiley: … pushed a new version that re-uses that.

**Pull request merged**